Ollamac/Views/ChatViews/AddChatView.swift

```diff
@@ -20,15 +20,13 @@ struct AddChatView: View {
     
     @State private var viewState: ViewState? = .loading
     
-    @State private var name: String = "New Chat"
```

Comment:
Hi @paramaggarwal, thanks for the PR with the idea of using the model's name as the default chat name. I get where you're coming from, but I'm going to stick with the current setup. Here's why:

The way things are now, even though it starts as "New chat," users can personalize the chat name based on what they're chatting about. This is super handy because it helps everyone keep track of what's going on in each chat. If we switch to using the model's name as the chat name by default, it might create some confusion.

Imagine you're in a bunch of chats with the same model but talking about different things. If they all have the same default name, it could get pretty tricky to figure out which chat is about what. So, even though we can change the chat name, starting with a model name might mean we have to rename chats more often to reflect the actual discussion topic.

Reply:
I see your point of encouraging people to customize the name of that chat. Hence, I agree that my suggestion of just using the model name is not a great solution. Maybe we could attempt doing what ChatGPT does and let the model itself generate a name for the chat.

The end goal is to have a meaningful name for the chat while also not making this a friction step in quickly starting a new chat.

Reply:
Got your point about sticking to the current setup for naming chats. Makes sense to avoid confusion with multiple chats using the same model. Keeping them as "New chat" does allow for better personalization and organization.

But, your PR got me thinking about the auto-naming feature, kind of like what ChatGPT does. It's not something we can roll out right now, but I'm interested in exploring this idea further. It could be a neat way to automatically give each chat a unique, relevant name, cutting down the need for manual renaming.